profiles/stehem.md

```diff
@@ -0,0 +1,6 @@
+# Stehem
+
+![](https://github.com/doesnotexist.png)
```

Comment:
This actually renders out to someone's profile. =) May want to still just add it to your username so it renders when you add it appropriately under https://github.com/settings